src/bert/sentence.py

```diff
@@ -12,6 +12,6 @@ def extract_keywords(self, top_n: int = 5) -> list:
             stop_words='english',
             top_n=top_n,
             use_mmr=True,
-            diversity=0.7
+            diversity=0.6
```

Comment:
why is the diversity changed here?

Reply:
it gives a better understanding and more the diversity more creative answers and less precise one , if we want a precise result we have to make it less to the point which is prefect for the code understanding